rclcpp/include/rclcpp/duration.hpp

```diff
@@ -106,6 +110,27 @@ class RCLCPP_PUBLIC Duration
   Duration
   operator*(double scale) const;
 
+  Duration &
+  operator*=(double scale);
+
+  Duration
+  operator*(const rclcpp::Duration & rhs) const;
+
+  Duration &
+  operator*=(const rclcpp::Duration & rhs);
+
+  Duration
+  operator/(double scale) const;
+
+  Duration &
+  operator/=(double scale);
+
+  Duration
+  operator/(const rclcpp::Duration & rhs) const;
+
+  Duration &
+  operator/=(const rclcpp::Duration & rhs);
```

Comment:
same here, it could return the scale instead?, yet i am not really sure the use case for the application.

Reply:
This PR was prompted because I was trying to refactor RobotTrajectory in MoveIt to use rclcpp::Duration objects to represent differences in time between states that previously were using `double` seconds to represent time. 

I remembered seeing division and having to do it with seconds to get it to compile. Looking back at the code though it is assigned into a double scale value.

Regardless, as you pointed out in your other comment, I grew the scope of the PR here making it more complex so I'm backing it back to much closer to the original intent. I removed the `operator/` stuff from this PR and the `seconds * seconds` stuff.